.github/workflows/build-lint-typecheck-test.yaml

```diff
@@ -35,6 +35,9 @@ jobs:
       - name: Typecheck
         run: yarn typecheck
 
+      - name: Build passport sample app
+        run: yarn workspace @imtbl/passport-sdk-sample-app build
```

Comment:
good job! treat this more seriously than before. passport sample app is important early indicators for us. 🙇

Reply:
thanks :) Yeah. The fact that this was missing gave me a false of security...